nautobot_circuit_maintenance/views.py

```diff
@@ -287,6 +293,15 @@ def get(self, request, *args, **kwargs):
             message += f": {mess_auth}"
         except ValueError as exc:
             message = str(exc)
+        except RedirectAuthorize as exc:
+            return redirect(
+                reverse(
+                    f"plugins:nautobot_circuit_maintenance:{str(exc.url_name)}",
```

Comment:
What happens if `exc.url_name` doesn't refer to a valid URL pattern? Do we need to catch the `NoReverseMatch` exception and handle it gracefully?

Reply:
Because the NotificationSource are defined by plugin configuration (user can't add or delete them), if we are at this point is because it is present, but adding a graceful handling will be good.